test/lib/usd/translators/UsdExportUVSetsFloatTest/UsdExportUVSetsTest_Float.ma

```diff
@@ -436,12 +352,10 @@ createNode mesh -n "SharedFacesCubeShape" -p "SharedFacesCube";
 	setAttr -s 3 ".uvst";
 	setAttr ".uvst[0].uvsn" -type "string" "map1";
 	setAttr ".uvst[1].uvsn" -type "string" "PairedFacesSet";
-	setAttr -s 23 ".uvst[1].uvsp[0:22]" -type "float2" 0 0 0.5 0 0.5 0.5
-		 0 0.5 0.5 0.5 1 0.5 1 1 0.5 1 0 0 0.5 0 0.5 0.5 0 0.5 0.5 0.5 1 0.5 1 1 0.5 1 0 0
-		 0.5 0 0.5 0.5 0 0.5 0.5 0.5 1 0.5 1 1;
+	setAttr -s 7 ".uvst[1].uvsp[0:6]" -type "float2" 0 0 0.5 0 0.5 0.5
+		 0 0.5 1 0.5 1 1 0.5 1;
 	setAttr ".uvst[2].uvsn" -type "string" "AllFacesSharedSet";
-	setAttr -s 24 ".uvst[2].uvsp[0:23]" -type "float2" 0 0 1 0 1 1 0 1 0
-		 0 1 0 1 1 0 1 0 0 1 0 1 1 0 1 0 0 1 0 1 1 0 1 0 0 1 0 1 1 0 1 0 0 1 0 1 1 0 1;
+	setAttr -s 4 ".uvst[2].uvsp[0:3]" -type "float2" 0 0 1 0 1 1 0 1;
```

Comment:
I would expect that these changes and the indexing changes just below would result in shorter arrays of UV values, but that doesn't seem to be the case. See more in testUsdExportUVSets.py below.

Reply:
From my experiments in Maya it looks like you should have removed some of the "mu" lines to simulate faces assigne to only one UV set.
- Create grid
- Copy map1 uv set
- Select and delete UVs in the UV editor
- Freeze history
When saved, a face assigned to 2 UV sets will look like:
```		f 4 128 131 -150 -130
		mu 0 4 67 68 79 78
		mu 1 4 1 2 8 7
```
While a single UV set face will be:
```		f 4 123 125 -145 -125
		mu 0 4 64 65 76 75
```

Reply:
What I see from experimenting in Maya is that there is one `mu` row per UV assigned to the face. The first number in a `mu` row being the UV set index. You might want to edit in Maya, freeze the edits, and then replicate the deleted rows you see in the edited file.